### ISSUE TITLE
make notebooks sortable

### ISSUE THREAD

**Issue by mtz** [orgzly/orgzly-android#19]
hi,

there is no possibility to sort notebooks on main overview page. 
perhaps you can make that possible!

thanks in advance

**Comment by nevenz** [comment]
You can order them by name or modification time (in Settings) currently.

Do you mean custom sort order, move them up and down manually?

**Comment by mtz** [comment]
Yes, i mean a manual sort or sort by priority as in the individual notes A-Z or for example very high, high, normal, low, ... .

But i think the manual sorting (drag'n'drop) would be for now the smartest solution.

**Comment by Chobbes** [comment]
I'd love to have this as well. I have a lot of org files (28 and counting...) and it gets a bit unwieldy at times :).

I'd even like to be able to kind of sort them in folders, or something.
